Source/BasicMathFunctions/muriscv_nn_elementwise_add_s8.c

```diff
@@ -147,8 +147,8 @@ muriscv_nn_status muriscv_nn_elementwise_add_s8(const int8_t *input_1_vect,
 
         int32_t sum = input_1 + input_2;
         sum = muriscv_nn_requantize(sum, out_mult, out_shift) + out_offset;
-        sum = __rv_max(sum, out_activation_min);
```

Comment:
I wanted to keep the original intrinsics inside the USE_PEXT defines for readability. Otherwise, one has to look up the macro first do understand what is happening under the hood (not that much work, but annoying). And we don't gain anything by using the macro here. But wdy think?

Reply:
The reason for this PR was just for consistency but your reasoning is fine. Let’s just close this, ok?

Reply:
Ok